src/bin/owi.ml

```diff
@@ -65,7 +65,7 @@ let workers =
   in
```

Comment:
Could you update the doc to say we use the total number of physical cores and not the value provided by Domain.recommended_domain_count anymore ?